lib/ReactViews/Custom/ChartCustomComponent.ts

```diff
@@ -105,9 +114,8 @@ export interface ChartCustomComponentAttributes {
  *                   or  `<chart>[["x","y","z"],[1,10,3],[2,15,9],[3,8,12],[5,25,4]]</chart>`.
  */
 
-type CatalogMemberType = Model<CatalogMemberTraits>;
 export default abstract class ChartCustomComponent<
-  CatalogItemType extends CatalogMemberType
+  CatalogItemType extends Chartable
 > extends CustomComponent {
```

Comment:
Had to make this `Chartable` to satisfy the types after refactoring `ChartExpandAndDownloadButtons` to `.tsx`